src/TBuffer.cpp

```diff
@@ -1256,6 +1256,65 @@ void TBuffer::translateToPlainText(std::string& incoming, const bool isFromServe
                     // We have manually disabled MXP negotiation
                     break;
 
+                case static_cast<quint8>('C'): {
+                    // A workaround for the ONE cursor movement command we CAN
+                    // emulate - the CUF Cursor forward one:
+                    // Needed for mud.durismud.com see forum message topic:
+                    // https://forums.mudlet.org/viewtopic.php?f=9&t=22887
+                    QString temp = QString(localBuffer.substr(localBufferPosition, spanEnd - spanStart).c_str());
```

Comment:
Can this be a faster `QStringRef`?

Reply:
Humm, yes, I do believe it could be...

Reply:
Ah, no it can't, but it can be a `QByteArray` which is a bit more lightweight I believe and still has a `toInt(bool*)` method - indeed I can use the `static QByteArray::fromRawData(...)` which does not copy the data from the std::string::substr(...) method's output.